pkg/apis/networking/types.go

```diff
@@ -35,13 +36,33 @@ type PodReference struct {
 	Namespace string
 }
 
+// ContainerPort represents a NamedPort on Pod.
+type ContainerPort struct {
```

Comment:
Do you think we can just call it NamedPort to make it more generic (I personally like to make NeworkPolicy and maybe other networking/policy generic for potential extensions later)?

Reply:
NamedPort sounds better to me as it could indicate it includes only named port, not all container ports.

Reply:
good idea